src/components/home.js

```diff
@@ -174,10 +198,32 @@ function Home(props) {
                     />
                   </div>
                 </div>
+
+                {window.innerWidth <= 769 && (
```

Comment:
Why 769? Is there any significance of this?

Reply:
This number is used by bootstrap as the minimum viewport size for mobile. As I thought to show the dropdown only for mobile devices I used this number.
<img width="722" alt="Screenshot 2020-04-12 at 9 17 42 AM" src="https://user-images.githubusercontent.com/17372422/79060061-79e5c600-7c9e-11ea-8b66-6b59144b014c.png">